app/models/role.rb

```diff
@@ -232,7 +232,13 @@ def self.override_search_operator(operator)
   private
 
   def sync_inheriting_filters
-    self.filters.where(:override => false).each { |f| f.inherit_taxonomies! }
+    self.filters.where(:override => false).each do |f|
+      unless f.save
+        errors.add :base, N_('One or more of the associated filters are invalid which prevented the role to be saved')
+        errors.empty?
```

Comment:
What does this line do? It seems like it has no effect?

Reply:
How can I create invalid filter ? -> filter with no permissions doesnt work